pkg/dynatrace/dynatrace.go

```diff
@@ -151,6 +151,27 @@ func (dynatraceApiClient *DynatraceApiClient) MakeRequest(method, path string, r
 	return dynatraceApiClient.httpClient.Do(req)
 }
 
+func (dynatraceApiClient *DynatraceApiClient) GetListDynatraceHttpMonitors(clusterId string) (*ExistsHttpMonitorInDynatraceResponse, error) {
```

Comment:
The naming conventions in this package are generally not the ones I'd use, but would it make sense to simplify this function's name to something like the following?
```suggestion
func (dynatraceApiClient *DynatraceApiClient) GetDynatraceHttpMonitors(clusterId string) (*ExistsHttpMonitorInDynatraceResponse, error) {
```